cmd/keytransparency-sequencer/server.go

```diff
@@ -26,9 +27,11 @@ import (
 	"google.golang.org/grpc"
 )
 
-func serveHTTPMetric(addr string) {
+func serveHTTPMetric(addr string, sqldb *sql.DB) {
```

Comment:
Please make "metrics" plural. Also, maybe make this name a bit more generic, e.g. `serveHTTPMetricsAndStatus`?